commitlint.config.ts

```diff
@@ -300,10 +300,18 @@ module.exports = {
                     let message = `Please do not exceed ${maxLineLength} characters in title (found ${headerLength}).`;
                     if (!headerStr.startsWith('Merge ') && headerLength > maxLineLength) {
                         offence = true;
+
+                        let colonIndex = headerStr.indexOf(':')
+
+                        let title = headerStr
+                        if (colonIndex > 0){
+                            title = headerStr.substring(colonIndex)
+                        }
+                        
                         let numRecomendations = 0;
-                        let lowerCaseHeaderStr = headerStr.toLowerCase()
+                        let lowerCaseTitle = title.toLowerCase()
```

Comment:
@realmarv missing semicolon in many places above ^

Reply:
Done